### PR TITLE
Small refactoring

### PR DESCRIPTION
**Pull request by Mikhail-M**
@dbarashev

### CODE REVIEW

File: demo/summer-2016.html [hunk 1]
```diff
@@ -211,8 +211,7 @@
 <script src="kitchen-sink/require.js"></script>
 <script>
 require.config({ paths : { "ace" : "../lib/ace" } });
-require(["ace/ace", "ace/ext/language_tools", "ace/ext/katex"], function(ace) {
-    langTools = ace.require("ace/ext/language_tools");
+require(["ace/ace", "ace/ext/language_tools", "ace/ext/katex"], function(ace, langTools) {
     katex = ace.require("ace/ext/katex");
```
Comment (dbarashev) [inline demo/summer-2016.html]: any sense to move katex into the function args too?

**Comment by dbarashev** [comment]
LGTM [1 comment]

File: demo/summer-2016.html (hunk 1 above)
Comment (Mikhail-M) [inline demo/summer-2016.html]: Sounds good

**Comment by Mikhail-M** [comment]
Could I merge it?

**Comment by dbarashev** [comment]
Yes, go ahead

**Pull request merged**